index.html

```diff
@@ -0,0 +1,484 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>Avengers | Creative Design Agency</title>
+    <meta name="description" content="Avengers, a Creative Design Agency based out of San Fransisco California, speicalizing in Web Design, UI Design, Icon Design, Branding & Logotype">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="css/main.css">
+  </head>
+  <body>
+    <input type="checkbox" id="mobileNavToggle" class="mobileNavToggle">
```

Comment:
No worries; like I said, off-canvas navigation is tricky.

Related -- do you really need your `id` and your `class` on this element to be identical? I'd probably just use the `id` here since it's pretty specific.